src/main/resources/META-INF/definitions.cnd

```diff
@@ -34,4 +34,4 @@
 [joant:facebookOAuthView] > jnt:content, jmix:authConnectorSettingView
 
 [joant:facebookButton] > jnt:content, joamix:oauthButtonConnector
- - buttonSize (string,choicelist[resourceBundle]) = 'large' < 'large', 'small', 'medium', 'xlarge'
+ - buttonSize (string,choicelist[resourceBundle]) = 'large' < 'large', 'small'
```

Comment:
is there any reason to remove these sizes?

Reply:
I removed because to have the same behaviour with the google button which only have small and large values.
It should not be a problem to remove these values, because the users can add a custom css class to customise the button